### PR TITLE
RHDEVDOCS-4766 Pipelines 1.8.2 Release Notes

### PR DESCRIPTION
**Pull request by sounix000**
- **Aligned team**: Dev Tools
- **OCP version for cherry-picking**: `enterprise-4.11`, `enterprise-4.12`
- **JIRA issues**: [RHDEVDOCS-4766 Pipelines 1.8.2 RN](https://issues.redhat.com/browse/RHDEVDOCS-4766)
- **Preview pages**: [Pipelines 1.8.2 RN](https://53178--docspreview.netlify.app/openshift-enterprise/latest/cicd/pipelines/op-release-notes.html#release-notes-1-8-2_op-release-notes) or [here](http://file.pnq.redhat.com/sosarkar/pipelines-1-8-2-rn/cicd/pipelines/op-release-notes.html#release-notes-1-8-2_op-release-notes)
- **SME review**: @rupalibehera / @vdemeester 
- **QE review**: @VeereshAradhya  / @ppitonak

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]

**Review by ppitonak — APPROVED** [review]

File: modules/op-release-notes-1-8.adoc [hunk 1]
```diff
@@ -609,4 +609,16 @@ config   1.8.1     True
...
+
+[id="release-notes-1-8-2_{context}"]
+== Release notes for {pipelines-title} General Availability 1.8.2
+
+With this update, {pipelines-title} General Availability (GA) 1.8.2 is available on {product-title} 4.10 and 4.11.
+
+[id="fixed-issues-1-8-2_{context}"]
+=== Fixed issues
+
+* Before this update, the `git-clone` task failed when we cloning a repository using SSH keys. With this update, the provision for nonroot user in the `git-init` task is removed, and the SSH program looks into the `$HOME/.ssh/` for the correct keys.
```
[the remark this answers is not in the document]
Comment (gabriel-rh) [reply modules/op-release-notes-1-8.adoc:622]: I'm just wondering if "provision" is the right word here - does if mean "need"/"requirement"?

**Review by gabriel-rh — COMMENTED** [review]
Some suggestions and a query about wording....

**Comment by rupalibehera** [comment]
Hey guys can we merge this PR 1.8.2 is shipped live cc @sounix000 @gabriel-rh

File: modules/op-release-notes-1-8.adoc [hunk 2]
```diff
@@ -609,4 +609,16 @@ config   1.8.1     True
...
+
+[id="release-notes-1-8-2_{context}"]
+== Release notes for {pipelines-title} General Availability 1.8.2
+
+With this update, {pipelines-title} General Availability (GA) 1.8.2 is available on {product-title} 4.10 and 4.11.
+
+[id="fixed-issues-1-8-2_{context}"]
+=== Fixed issues
+
+* Before this update, the `git-clone` task failed when cloning a repository using SSH keys. With this update, the role of the non-root user in the `git-init` task is removed, and the SSH program looks in the `$HOME/.ssh/` for the correct keys.
```
Comment (Preeticp) [inline modules/op-release-notes-1-8.adoc:622]: s/in the `$HOME/.ssh/` for the correct keys. /in the `$HOME/.ssh/` directory/folder for the correct keys.

**Review by Preeticp — APPROVED** [review]

**Pull request merged**